src/data/faq.json

```
@@ -429,7 +446,7 @@
                         ]
                     },
                     {
-                        "title": "If I see a high-risk encounter in the App, i.e. a red card, should I do a rapid antigen test or PCR-Test?",
+                        "title": "If I see a high-risk encounter in the App, i.&nbsp;e. a red card, should I do a rapid antigen test or PCR-Test?",

```

```suggestion
                        "title": "If I see a high-risk encounter in the App, i.e. a red card, should I do a rapid antigen test or PCR-Test?",
```
remove `&nbsp;` from title (doesn't work and isn't correct in an English abbreviation)